its/ruling/src/test/resources/expected/python-S2638.json

```diff
@@ -73,6 +70,9 @@
 151,
 193,
 ],
+'project:django-2.2.3/django/utils/text.py':[
+284,
```

Comment:
Somewhat of an FP. So the detection is correct here read should have a second parameter called size but the message says the parameter name is `null`

Reply:
I guess this is due to the fact the parameter starts with `__` ?
We should probably have a small ticket to fix this and update the message in such case.
It would be nice to have this case as a unit test case in S2638 unit tests.

Reply:
Not sure about the `__`, the parameter seems to be called `size`. Still [I have created this issue to investigate.](https://sonarsource.atlassian.net/browse/SONARPY-1363)